astropy/modeling/tests/test_parameters.py

```diff
@@ -472,7 +469,8 @@ def test__set_unit(self):
         with pytest.raises(ValueError) as err:
             param._set_unit(u.K)
         assert str(err.value) == \
-            "Cannot change the unit attribute directly, instead change the parameter to a new quantity"
+            "Cannot change the unit attribute directly, instead change the " + \
```

Comment:
I would prefer `\` is gotten rid of using parenthesis.

Reply:
Done in 10d5232